.github/workflows/analysis.yml

```diff
@@ -0,0 +1,50 @@
+name: Analysis
```

Comment:
Maybe the name could be more specific? Like MSVC Analysis?

Reply:
Same for the file name.

Reply:
If you look at the checks running below this combines with the Job name: `Analysis / Microsoft C++ Code Analysis`. 

Also you can also put multiple jobs and/or matrices in a workflow file which is why I adjusted the files to be more generic like a category and use the job name to nail down the specifics.

Reply:
Fair enough :)